### PR TITLE
Add OpenGov to Darwinia

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by boundless-forest** [comment]
@AurevoirXavier Please update the branch to pass CI

**Review by boundless-forest — COMMENTED** [review]
LGTM. Lacking the support for the evm, I'll raise a PR after this merge.

**Review by boundless-forest — APPROVED** [review]

File: runtime/darwinia/src/pallets/governance.rs [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	type OnSlash = Treasury;
+	type PalletId = pallet_config::TreasuryPid;
+	type ProposalBond = ProposalBond;
+	type ProposalBondMaximum = ();
+	type ProposalBondMinimum = ConstU128<DARWINIA_PROPOSAL_REQUIREMENT>;
+	type RejectOrigin = RootOrAll<CouncilCollective>;
+	type RuntimeEvent = RuntimeEvent;
+	type SpendFunds = ();
+	type SpendOrigin = frame_support::traits::NeverEnsureOrigin<Balance>;
+	type SpendPeriod = ConstU32<{ 7 * DAYS }>;
```
Comment (AurevoirXavier) [inline runtime/darwinia/src/pallets/governance.rs:135]: 28 DAYS

File: runtime/darwinia/src/pallets/governance.rs [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+	type Preimages = Preimage;
+	type RuntimeCall = RuntimeCall;
+	type RuntimeEvent = RuntimeEvent;
+	type Scheduler = Scheduler;
+	type Slash = Treasury;
+	type SubmissionDeposit = ConstU128<{ DARWINIA_PROPOSAL_REQUIREMENT }>;
+	type SubmitOrigin = frame_system::EnsureSigned<AccountId>;
+	type Tally = pallet_conviction_voting::TallyOf<Self>;
+	type Tracks = TracksInfo;
+	type UndecidingTimeout = ConstU32<{ 14 * DAYS }>;
```
Comment (AurevoirXavier) [inline runtime/darwinia/src/pallets/governance.rs:96]: 28 DAYS

**Pull request merged**